java edot skip certificate doc (#358)
elastic/opentelemetry: docs/reference/edot-sdks/java/configuration.md

Setting `OTEL_JAVAAGENT_LOGGING=application` will disable EDOT agent logging feature and attempt to use the application logger.
As [documented here in the upstream documentation](https://opentelemetry.io/docs/zero-code/java/agent/configuration/#java-agent-logging-output),
support for this depends on the application and logging libraries used.

## Exporter certificate verification

The EDOT Java agent provides the ability to toggle the exporter endpoint certificate verification with the `ELASTIC_OTEL_VERIFY_SERVER_CERT` configuration option (`true` by default).

When the endpoint certificate is not trusted by the JVM where the agent runs, the common symptom is security-related exceptions with the following message: `unable to find valid certification path to requested target`.

This is common in the following scenarios:

> - when endpoint uses a self-signed certificate not trusted by the JVM
> - when the certificate authority used by the endpoint certificate is not trusted by the JVM

Reviewer: ```suggestion
- When endpoint uses a self-signed certificate not trusted by the JVM
- When the certificate authority used by the endpoint certificate is not trusted by the JVM
```

> The usual solution is to add the certificate or certificate authority to the JVM trust store, which requires to modify the JVM trust store.

Reviewer 2: ```suggestion
The usual solution is to add the certificate or certificate authority to the JVM trust store, which requires modifying the JVM trust store.
```

> The usual solution is to add the certificate or certificate authority to the JVM trust store, which requires modifying the JVM trust store.
> When this trust store modification is not possible or not practical, for example when troubleshooting or with a local deployment, certificate verification can be disabled by setting `ELASTIC_OTEL_VERIFY_SERVER_CERT` to `false`.
> 
> This however need to be evaluated carefully as it lowers the communication security and could allow for man-in-the-middle attacks where the data could be intercepted between the agent and the collector endpoint.

Reviewer: Some minor adjustments to the text, and restructuring because I _think_ line 176 is referring to line 174. If that's the case, I think it makes sense to keep those in the same paragraph. 

```suggestion
One solution is to add the certificate or certificate authority to the JVM trust store, which requires modifying the JVM trust store.

If trust store modification is not possible or not practical, for example when troubleshooting or working with a local deployment, certificate verification can be disabled by setting `ELASTIC_OTEL_VERIFY_SERVER_CERT` to `false`. This however need to be evaluated carefully as it lowers the communication security and could allow for man-in-the-middle attacks where the data could be intercepted between the agent and the collector endpoint.
```
